.gitignore

```diff
@@ -27,6 +27,7 @@
 .scalastyle
 .settings
 .cache*
+.tmpBin
```

Comment:
I'm not sure what comes from the IDE and what is from sbt, but I think this one is from the IDE. I talked to one of the IDE developers at ScalaDays and said that it would be nice if all those "temporary" files could be located in **one** directory.